test/layers/heteroconv.jl

```diff
@@ -33,4 +33,62 @@
                                 (:B, :to, :A) => GraphConv(64 => 32, relu));
         @test length(layer.etypes) == 2
     end
+
+    @testset "Destination node aggregation" begin
+        # deterministic setup to validate the aggregation
+        d, n = 3, 5
+        g = GNNHeteroGraph(((:A, :to, :B) => ([1, 1, 2, 3], [1, 2, 2, 3]),
+                (:B, :to, :A) => ([1, 1, 2, 3], [1, 2, 2, 3]),
+                (:C, :to, :A) => ([1, 1, 2, 3], [1, 2, 2, 3])); num_nodes = Dict(:A => n, :B => n, :C => n))
+        model = HeteroGraphConv([
+                (:A, :to, :B) => GraphConv(d => d, init = ones, bias = false),
+                (:B, :to, :A) => GraphConv(d => d, init = ones, bias = false),
+                (:C, :to, :A) => GraphConv(d => d, init = ones, bias = false)]; aggr = +)
+        x = (A = rand(Float32, d, n), B = rand(Float32, d, n), C = rand(Float32, d, n))
+        y = model(g, x)
+        weights = ones(Float32, d, d)
+
+        ### Test default summation aggregation
+        # B2 has 2 edges from A and itself (sense check)
+        expected = sum(weights * x.A[:, [1, 2]]; dims = 2) .+ weights * x.B[:, [2]]
+        output = y.B[:, [2]]
+        @assert expected ≈ output
```

Comment:
all these `@assert` should become `@test`

Reply:
ha! Apologies, I was lazy to import Test when developing in my scratch file and then forgot to replace 🤦‍♂️